src/integrationTest/resources/application-intTest.yaml

```diff
@@ -43,6 +43,11 @@ darts:
     trim-workspace: ${java.io.tmpdir}/audiotransform/trim
     re-encode-workspace: ${java.io.tmpdir}/audiotransform/encode
     temp-blob-workspace: ${java.io.tmpdir}/audiotransform/tempworkspace
+
+  transformation:
+    service:
+      audio:
+        file:
```

Comment:
should this be blank?

Reply:
yes most the tests wont need to use an actual audio file.